### ISSUE TITLE
Does not start with windows

### ISSUE THREAD

**Issue by charmarkk** [White-Tiger/T-Clock#139]
Despite all my efforts, I can't get T-Clock to relaibly start with windows. I've tried running as admin, moving the program, setting startup settings in CCleaner and other managers - nothing.

**Comment by White-Tiger** [comment]
how does this unreliability manifest itself?

**Comment by charmarkk** [comment]
The program simply doesn’t open; it’s not present in task manager and I only see Windows’ clock until I manually start the program.

**Comment by White-Tiger** [comment]
well, if that's the case, I wonder why (and actually how) your tries to manually setup the startup settings in CCleaner and others could have failed as well... what's the difference between manually running it and having it run on startup?

Some part of T-Clock has to be running though. Either a zombie process or an error message. Even if it crashes for some reason, you should be able to see an entry in Windows event log.
However, you might try the build I've posted here: https://github.com/White-Tiger/T-Clock/issues/118#issuecomment-368132437
I've changed some hooking code and it'll create a log file on your desktop which might help as well.

It might also help to know more about your PC's software setup. _(OS, bitness, Windows updates, software that modifies parts of Windows etc.)_

**Comment by charmarkk** [comment]
I'll try that version, thanks. It seems nothing is in Windows' log to report any errors about it, but it definitely hasn't been running any process when I haven't seen it running in the taskbar.

I _do_ have it set to run at startup now (just to verify), and I can come back when I notice that it hasn't run - might have something to do with sleep.  

![image](https://user-images.githubusercontent.com/22264269/36951818-a550f984-1fcd-11e8-8a21-39495292657e.png)

I use Win10 64-bit, 1709 build 16299.248. The only "major" thing I have that "modifies" Windows is Rainmeter and Rocketdock, but these are both applications and don't change anything inherent about Windows.

**Comment by White-Tiger** [comment]
did you reboot your Windows yet?

**Comment by charmarkk** [comment]
Yep, it seems to have persisted this time. I don’t have access to my machine at the moment, but I’ll check again when I can and update with more details!

**Comment by charmarkk** [comment]
Just restarted for updates - settings left as above. T-Clock hasn't started and no, the process is not running.

**Comment by charmarkk** [comment]
Odd - it's actually just not there! And that's with the settings mentioned earlier!

**Comment by White-Tiger** [comment]
does T-Clock still think that it'll start with Windows? So what does the about dialog say?

**Comment by charmarkk** [comment]
![image](https://user-images.githubusercontent.com/22264269/37256237-d77a7fb8-2525-11e8-8213-d82775a4d5b7.png)

Exact same as before/

**Comment by White-Tiger** [comment]
so neither your Task Manager's "Startup" tab, nor the folder
`%AppData%\Microsoft\Windows\Start Menu\Programs\Startup` _(you can Copy&Paste this into the File Explorer address line)_
have anything with T-Clock in there?

**Comment by charmarkk** [comment]
Ah. Okay, so there is a shortcut, but its for an old, nonexistent file. Let me try deleting that, rechecking the setting, and restarting.

**Comment by MartinD55** [comment]
This helped me get T-Clock to start reliably on Windows 10.
A.) How can I stop the logging now?
B.) Why don't later versions act this way?

**Comment by White-Tiger** [comment]
what exactly did you do or had been the problem?
A) if you're using a log enabled build, simply uncheck the logging functionality at the "about" screen
B) if you're referring to this issue, the latest build (that came to be after the fix) is indeed the testing build with logging. However it's not the one from #118 mentioned above, that's actually older than this fix.

**Comment by MartinD55** [comment]
Hello-
Long time T-Clock user.
Updated my work PC to Windows 10. T-clock would not start after. I had been using RC x64 v2.4.4 build 492.
What build should I be using?

**Comment by blackcrack** [comment]
you should not past the whole Mail from your mail account, you need only answer in this tread :
"Reply to this email directly, [**_view it on GitHub_**](https://github.com/White-Tiger/T-Clock/issues/139?email_source=notifications&email_token=AB4O7VMGIOV4HZ72C4PHUJDRGAXETA5CNFSM4ETMSXM2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEN7DVSI#issuecomment-595475145), or unsubscribe."

2nd this is the Actual Release https://github.com/White-Tiger/T-Clock/releases 

there:   
https://github.com/White-Tiger/T-Clock/

(2nd line commits=>branches=>Packages=> **RELEASES**)

 it's not start, klick the F... nice Clock on your Systtray, open with right Mouseklick the Lists and klick T-Clock Options, check the
 [ ]  **Start T-Clock Redux when Windows starts**

it's off, then klick it, if not, klick it off and okey, later open the Options again and check it on again.. then test it by reboot..

edit: and if it's not works, kick your Computer out of the f.. Window and take a new Release of your Computer, if this not works (kickin' out of the Window) and the Window have a too small Release number from your House .. Update your Window from your House ! \*bg*
